components/Downloads/DownloadReleasesTable.tsx

```diff
@@ -2,55 +2,59 @@ import { FormattedMessage } from 'react-intl';
+import { useNodeReleasesData } from '../../hooks/useNodeReleasesData';
 import type { FC } from 'react';
 
-type DownloadReleasesTableProps = { releases: ExtendedNodeVersionData[] };
+type DownloadReleasesTableProps = {};
 
-const DownloadReleasesTable: FC<DownloadReleasesTableProps> = ({
-  releases,
-}) => (
-  <table id="tbVersions" className="download-table full-width">
-    <thead>
-      <tr>
-        <td>Version</td>
-        <td>LTS</td>
-        <td>Date</td>
-        <td>V8</td>
-        <td>npm</td>
-        <td>
-          NODE_MODULE_VERSION<Link href="#ref-1">[1]</Link>
-          <span id="backref-1"></span>
-        </td>
-        <td></td>
-      </tr>
-    </thead>
-    <tbody>
-      {releases.map((release, key) => (
-        <tr key={key}>
-          <td data-label="Version">Node.js {release.nodeNumeric}</td>
-          <td data-label="LTS">{release.ltsName}</td>
-          <td data-label="Date">
-            <time>{release.releaseDate}</time>
-          </td>
-          <td data-label="V8">{release.v8}</td>
-          <td data-label="npm">{release.npm}</td>
-          <td data-label="NODE_MODULE_VERSION">{release.modules}</td>
-          <td className="download-table-last">
-            <a href={`https://nodejs.org/download/release/${release.node}`}>
-              <FormattedMessage id="components.downloadReleasesTable.releases" />
-            </a>
-            <a href={getNodejsChangelog(release.node)}>
-              <FormattedMessage id="components.downloadReleasesTable.changelog" />
-            </a>
-            <a href={getNodeApiLink(release.node)}>
-              <FormattedMessage id="components.downloadReleasesTable.docs" />
-            </a>
+const DownloadReleasesTable: FC<DownloadReleasesTableProps> = () => {
+  const { releases } = useNodeReleasesData();
+
+  return (
+    <table id="tbVersions" className="download-table full-width">
+      <thead>
+        <tr>
+          <td>Version</td>
+          <td>LTS</td>
+          <td>Date</td>
+          <td>V8</td>
+          <td>npm</td>
+          <td>
+            NODE_MODULE_VERSION<Link href="#ref-1">[1]</Link>
+            <span id="backref-1"></span>
           </td>
+          <td></td>
         </tr>
-      ))}
-    </tbody>
-  </table>
-);
+      </thead>
+      <tbody>
+        {releases.map(release => (
+          <tr key={release.major}>
+            <td data-label="Version">Node.js {release.version}</td>
+            <td data-label="LTS">{release.codename}</td>
+            <td data-label="Date">
+              <time>{release.releaseDate}</time>
+            </td>
+            <td data-label="V8">{release.v8}</td>
+            <td data-label="npm">{release.npm}</td>
+            <td data-label="NODE_MODULE_VERSION">{release.modules}</td>
+            <td className="download-table-last">
+              <a
+                href={`https://nodejs.org/download/release/${release.versionWithPrefix}`}
```

Comment:
```suggestion
                href={`/${release.versionWithPrefix}`}
```

Would this also work? Since the website is already nodejs.org